static/app/components/replays/replayRageClickSdkVersionBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled';
+
+import replaysDeadRageBackground from 'sentry-images/spot/replay-dead-rage-changelog.svg';
+
+import {Button} from 'sentry/components/button';
+import PageBanner from 'sentry/components/replays/pageBanner';
+import {IconBroadcast} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {MIN_DEAD_RAGE_CLICK_SDK} from 'sentry/utils/replays/sdkVersions';
+import useDismissAlert from 'sentry/utils/useDismissAlert';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export default function ReplayRageClickSdkVersionBanner() {
+  const organization = useOrganization();
+
+  const {dismiss, isDismissed} = useDismissAlert({
+    key: `${organization.id}:replay-rage-dead-click-sdk-version-banner-v1`,
+  });
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <PageBanner
+      button={<Button priority="primary">Read Changelog</Button>}
+      description={t(
+        "Understand what your users do when your user experience doesn't meet their expectations"
+      )}
+      heading={t('Introducing Rage and Dead Clicks')}
+      icon={<IconBroadcast size="sm" />}
+      image={replaysDeadRageBackground}
+      onDismiss={dismiss}
+      title={tct("What's new in [version]", {
+        version: (
+          <PurpleText>
+            {tct(`version [version_number]`, {
+              version_number: MIN_DEAD_RAGE_CLICK_SDK,
```

Comment:
Should we change the version here to reflect latest version (7.69.0)? Though it will probably get too confusing having all these different versions around, especially since latest version will get out of sync very quickly.

Reply:
We could work the banner to show the minimum version, as well as the latest?

I think the minimum version is the new/important bit to show. Folks can look at their own code to see where they are currently, and all the other docs and github repo are all wired up to show the latest. Installing the latest is the easy path if you're starting from google.

Reply:
Yeah let's leave this 👍